charts/jenkins/README.md

```diff
@@ -149,15 +149,22 @@ In order to upgrade, [uninstall](#uninstall-chart) the Jenkins Deployment before
 See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), or run these configuration commands:
 
 ```console
-# Helm 2
-$ helm inspect values jenkinsci/jenkins
-
 # Helm 3
 $ helm show values jenkinsci/jenkins
+
+# Helm 2
+$ helm inspect values jenkinsci/jenkins
 ```
 
+For a summary of all configurable options, see [VALUES_SUMMARY.md](./VALUES_SUMMARY.md)
 
-### Mounting Volumes into Your Agent Pods
+### Allow Limited HTML Markup in User-Submitted Text
+Some third-party systems (e.g. GitHub) use HTML-formatted data in their payload sent to a Jenkins webhook (e.g. URL of a pull-request being built).
+To display such data as processed HTML instead of raw text set `master.enableRawHtmlMarkupFormatter` to true.
+This option requires installation of the [OWASP Markup Formatter Plugin (antisamy-markup-formatter)](https://plugins.jenkins.io/antisamy-markup-formatter/).
+This plugin is **not** installed by default and may be added to `master.additionalPlugins`.
```

Comment:
the original text:
> Some third-party systems, e.g. GitHub, use HTML-formatted data in their payload sent to a Jenkins webhooks, e.g. URL of a pull-request being built. To display such data as processed HTML instead of raw text set `master.enableRawHtmlMarkupFormatter` to true. This option requires installation of OWASP Markup Formatter Plugin (antisamy-markup-formatter). The plugin is **not** installed by default, please update `master.installPlugins`.